week1/main.css

```diff
@@ -0,0 +1,179 @@
+/* This is main styling of the page */
+body{
+  background-color: white;
+  font-size: large;
+  font-family: 'Times New Roman';
+}
+p{
+  font-size: 30px;
+} 
+/* This is main styling end */
+
+/*Add-head-logo styling*/
+.nav-bar-top{
+  border-style:none;
+  border-radius: 5px;
+  border-color: #bab5d4;
+  border-width: thick;
+  background-color: #a2a4ac;
+  box-shadow: 15px 15px 15px #888888;
+  text-decoration: none;
+}
+
+/*Add-head-logo styling end*/
+
+/* Brand-logo styling*/
+.brand-logo img{
+  border-radius: 50%;
+  border: 5px solid #59baf2;
+  padding: 0px;
+  border-color: #4a4a4a;
+  margin-left: auto;
+  margin-right: auto
+}
+/* Brand-logo styling finished*/
+
+/*Navigation bar styling*/
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #505045;
+  border-radius: 5px;
+  border-style:hidden;
+  box-shadow: 15px 15px 15px #888888;
+  text-decoration: none;
+  
+}
+
+li {
+  float: left;
+}
+
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  font-size: 30px;
+  text-shadow: 5px 5px 10px blue"
+}
+
+li a:hover:not(.active) {
+  background-color: #101111;
+}
+
+.active {
+  background-color: #925129;
+}
+a:link, a:visited{
+  text-decoration:none;
+}
+/*Navigation bar end*/
+
+/* Blog-main styling*/
+.blog-main{
+  border-style:hidden;
+  border-radius: 8px;
+  border-width: 3px;
+  border-color:#101111;
+  padding: 20px;
+  margin: 5px;
+  background-color: #f5fdce;
+  box-shadow: 5px 10px 10px #888888;
+}
+
+h1{
+  text-decoration: underline;
+  padding: 20px;
+  margin: 5px;    
+  text-align: center;
+}
+/* Blog-main styling finished*/
+
+/* Reference P styling*/
+.blog-main p:last-of-type{
+  font-size: 15px;
+  font-family: sans-serif;
```

Comment:
Generally speaking, you don't want to mix serif fonts (like Times New Roman) and sans-serif fonts on the same page. It's rarely something you see on modern webpages and it can give a 'cheap' look to your site no matter how much effort you put in.